aodndata/moorings/products_handler.py

```diff
@@ -139,6 +141,20 @@ def _read_manifest(self):
                 )
             self.products_to_create = set(manifest['products'])
 
+        # Even if only the gridded product is explicitly requested, we need to re-generate and publish the hourly too,
+        # as it is the input file for the gridded.
+        if 'gridded' in self.products_to_create:
+            self.products_to_create.add('hourly')
+
+    @property
```

Comment:
I would change the name to something like "input/output options " or alike to raise awareness that it's just IO options.

Reply:
They're not all "input/output options" (the "*_url_prefix" args end up written to variable attributes in the output files).
The one thing they all have in common is that all the product-generating functions accept these args, so I thought this name was fairly descriptive.